recipes/optiwindnet/recipe.yaml

```diff
@@ -0,0 +1,100 @@
+context:
+  version: 0.1.1
+  python_min: 3.11
+  # on upgrade, update the sha256 hash
+
+package:
+  name: optiwindnet
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.windenergy.dtu.dk/TOPFARM/OptiWindNet/-/archive/v${{ version }}/OptiWindNet-v${{ version }}.tar.bz2
+  sha256: 90ec0036dba9dde4a3789a2759937f34c16f7f0f0d6820b61ab6a8ea124e3e05
+
+build:
+  number: 0
+  noarch: python
+  script:
+    env:
+      SETUPTOOLS_SCM_PRETEND_VERSION: ${{ version }}
```

Comment:
If the package defines the version correctly you do not need this. If there is a problem with it, please report upstream.